### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by aisharina99**
ready

### CODE REVIEW

**Comment by aisharina99** [comment]
ready

**Review by ghost — COMMENTED** [review]
You have 2 commit, please combine it into 1

**Comment by aisharina99** [comment]
ready

File: app/views/users/new.html.erb [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
-      <%= f.email_field :email %>
-      <%= f.label :password %>
-      <%= f.password_field :password %>
-      <%= f.label :password_confirmation, t("confirmation") %>
-      <%= f.password_field :password_confirmation %>
-      <%= f.submit  "Create my account", class: "btn btn-primary" %>
-    <% end %>
-  </div>
+        </ul>
+    </div>
```
Comment (ngocvt-0484) [inline app/views/users/new.html.erb]: wrong indent

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ngocvt-0484) [inline app/helpers/static_pages_helper.rb]: delete this file 
because it empty

**Comment by aisharina99** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] =  "Invalid email/password combination"
+      render :new
+    end
+  end
+
+  def destroy
+    log_out
```
Comment (huongnt-2545) [inline app/controllers/sessions_controller.rb]:
```suggestion
    log_out if logged_in?
```

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -1,11 +1,26 @@
-  <header class="navbar navbar-fixed-top navbar-inverse">
+<header class="navbar navbar-fixed-top navbar-inverse">
     <div class="container">
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb]: wrong the indentation from here, please fix
note: indent = 2 spaces
```suggestion
  <div class="container">
     <%= link_to "sample app", "#", id: "logo" %>
     ..
```

File: test/controllers/static_pages_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/static_pages_controller_test.rb]: delete this file, it's not useful

**Review by huongnt-2545 — COMMENTED** [review]
Why you delete the migration files??

File: app/controllers/sessions_controller.rb (hunk 3 above)
Comment (aisharina99) [reply app/controllers/sessions_controller.rb]: I delete migration files because I have made new migration file.

**Comment by aisharina99** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull

**Pull request merged**